### DISCUSSION TITLE
Line breaks on the napari hub different to github

### DISCUSSION THREAD

**Discussion by haesleinhuepf** [chanzuckerberg/napari-hub#378] [General]
Hi all,

I just noticed that napari hub plugin pages deal with line breaks (between badges) differently compared to github. The layout does not look good on the hub:

https://www.napari-hub.org/plugins/napari-skimage-regionprops

![image](https://user-images.githubusercontent.com/12660498/148036840-60dc5c39-060d-42df-a092-4d22cbe0e16b.png)

https://github.com/haesleinhuepf/napari-skimage-regionprops

![image](https://user-images.githubusercontent.com/12660498/148036940-dd097997-1b74-4e3d-a67e-00168cb9a329.png)

Also line breaks within markdown text shows on the hub like this:

![image](https://user-images.githubusercontent.com/12660498/148037330-2665a7b8-ccb9-4a43-b567-5b10d84ad002.png)

and on github like that:

![image](https://user-images.githubusercontent.com/12660498/148037373-d3fd80f2-0d76-42fe-b013-b39656a8ce5c.png)

Is this something we plugin developers should deal with or can this be fixed on hub side?

Thanks!

Best,
Robert

**Comment by neuromusic** [comment]
You're right @haesleinhuepf , this is a bug in our markdown rendering. I've created an issue here: https://github.com/chanzuckerberg/napari-hub/issues/418

**Comment by neuromusic** [comment]
actually @haesleinhuepf , there seems to be something else going on here. there are lots of plugins with line breaks that render fine 🤔 

do you do your development on windows by chance?
